src/pip/_internal/configuration.py

```diff
@@ -117,29 +118,21 @@ def __init__(self, isolated, load_only=None):
         # type: (bool, Kind) -> None
         super(Configuration, self).__init__()
 
-        _valid_load_only = [kinds.USER, kinds.GLOBAL, kinds.SITE, None]
-        if load_only not in _valid_load_only:
+        if load_only not in VALID_LOAD_ONLY:
             raise ConfigurationError(
                 "Got invalid value for load_only - should be one of {}".format(
-                    ", ".join(map(repr, _valid_load_only[:-1]))
+                    ", ".join(map(repr, VALID_LOAD_ONLY[:-1]))
```

Comment:
I’d probably prefer `(kind for kind in VALID_LOAD_ONLY if kind)`. I needed to dart between the definition and usage to understand what the `[:-1]` part is about.

Alternatively we should remove `None` from `VALID_LOAD_ONLY` and add a `or load_only is None` at line 121.

Reply:
I'd go for the alternative, that also seems clearer to me.